includes/migration-guide/runtime/asp/aspnet-validationcontextmembername-not-null-when-using-custom.md

```diff
@@ -2,8 +2,8 @@
 
 |   |   |
 |---|---|
-|Details|In .NET Framework 4.7.2 and earlier versions, when using a custom <xref:System.ComponentModel.DataAnnotations.ValidationAttribute?displayProperty=nameWithType>, the <xref:System.ComponentModel.DataAnnotations.ValidationContext.MemberName?displayProperty=nameWithType> property returns <code>null</code>.  In .NET Framework 4.8, it returns the member name.|
-|Suggestion|To restore the previous behavior, add the following setting to your app config file:<pre><code class="lang-xml">&lt;configuration&gt;&#13;&#10;&lt;appSettings&gt;&#13;&#10;...&#13;&#10;&lt;add key=&quot;aspnet:GetValidationMemberName&quot;  value=&quot;true&quot;/&gt;&#13;&#10;...&#13;&#10;&lt;/appSettings&gt;&#13;&#10;&lt;/configuration&gt;&#13;&#10;</code></pre>This behavior will change in an upcoming service release, when you will have to explicitly opt in to the new behavior. The property will return a non-null value for a custom `ValidationAttribute` if the `aspnet:GetValidationMemberName` switch is set to `true`.|
+|Details|In .NET Framework 4.7.2 and earlier versions, when using a custom <xref:System.ComponentModel.DataAnnotations.ValidationAttribute?displayProperty=nameWithType>, the <xref:System.ComponentModel.DataAnnotations.ValidationContext.MemberName?displayProperty=nameWithType> property returns <code>null</code>.  Starting with [.NET Framework October 2019 Preview of Quality Rollup](https://devblogs.microsoft.com/dotnet/net-framework-october-2019-preview-of-quality-rollup/) for .NET Framework 4.8, it could return the member name after adding the following settings in the web config file:|
```

Comment:
|Details|In .NET Framework 4.7.2 and earlier versions, when using a custom <xref:System.ComponentModel.DataAnnotations.ValidationAttribute?displayProperty=nameWithType>, the <xref:System.ComponentModel.DataAnnotations.ValidationContext.MemberName?displayProperty=nameWithType> property returns <code>null</code>.  Starting with [.NET Framework October 2019 Preview of Quality Rollup](https://devblogs.microsoft.com/dotnet/net-framework-october-2019-preview-of-quality-rollup/) for .NET Framework 4.8, it returns the member name when using the following settings in the *web.config* file:|

Reply:
should this be inside a suggestion Rick?

Reply:
I missed the suggestion. I've never seen suggestion. I don't think so. What should be there?

https://review.docs.microsoft.com/en-us/dotnet/framework/migration-guide/runtime/4.0-4.8?branch=pr-en-us-15814#aspnet-validationcontextmembername-is-not-null-when-using-custom-dataannotationsvalidationattribute

Reply:
updated

Reply:
This is the suggestion button just for reference. Then the person can just commit your suggestion.

![image](https://user-images.githubusercontent.com/12971179/68705090-4d3c4f80-0542-11ea-9e1b-3dea6d77f30d.png)

Reply:
I see, I didn't see the commit button for some reason so I submitted the change.